ncnews-9

SonnyH98 opened the pull request:
Inside of the view/comments button I have added an input section where you can add a comment.
If you insert a valid comment e.g. having the name as a valid user, you get a browser alert telling you to press okay to refresh the page to prevent the user from sending the same comment twice.
If the comment is invalid you get an alert to check the name of the comment to ensure it is a valid user

Commit:
ncnews-9

dappernerddesigns reviewed (changes requested):
Please make sure to remove console logs you don't need :)

When a user is offline/connection issue they see an alert telling them their user name is wrong. This is not ideal at explaining what is going on if the username is correct.

I would like you to remove the alerts as they are not great for accessibility nor do they perform well on mobile. You can create error texts and use state to hide or show them accordingly.

Your text input is currently an input box, this is designed for a single line of text. We would expect commenters to be able to add lots of text. Look up the differences between an input or a text area.

When you have addressed the alerts and input please resubmit.

Commit:
Responded to pull request comments

SonnyH98 commented:
Removed console.logs
Changed input box to textarea for comment box.
Replaced alerts with a conditionally rendered comment depending on if the comment was successful or not using useState.

Commit:
changed some code
